### PR TITLE
[Helm] Updated kube-state-metrics cluster role with missing resources

### PR DESCRIPTION
**Pull request by pmontanari**
Added missing configmaps + secrets in clusterRole

### CODE REVIEW

File: helm/exporter-kube-state/templates/clusterrole.yaml [hunk 1]
```diff
@@ -24,25 +26,52 @@ rules:
...
+  - watch
+- apiGroups:
+  - autoscaling
   resources:
   - horizontalpodautoscalers
-  verbs: ["list", "watch"]
+  verbs:
+  - list
+  - watch
+- apiGroups:
```
Comment (gianrubio) [inline helm/exporter-kube-state/templates/clusterrole.yaml]: I'm wondering where does this values come from? I didn't see it in the kube-state-metrics repo

Comment (brancz) [reply helm/exporter-kube-state/templates/clusterrole.yaml]: it's for the kube-rbac-proxy sidecar

Comment (pmontanari) [reply helm/exporter-kube-state/templates/clusterrole.yaml]: Btw, the kube-rbac-proxy sidecar is missing in the helm chart.

Comment (gianrubio) [reply helm/exporter-kube-state/templates/clusterrole.yaml]: @pmontanari can you also add kube-rbac-proxy in this PR?

Comment (pmontanari) [reply helm/exporter-kube-state/templates/clusterrole.yaml]: Ok i'll do

Comment (pmontanari) [reply helm/exporter-kube-state/templates/clusterrole.yaml]: Well... There are a lot of differences between Helm version and  "standard" version: PodSecurityPolicies, namespaceSelector,...
I don't feel like updating all of it... sorry

Comment (rogermakram) [reply helm/exporter-kube-state/templates/clusterrole.yaml]: Any update on this PR? Without these resources in the cluster role, I am getting a lot of noise in the logs
example: reflector.go:205] k8s.io/kube-state-metrics/collectors/collectors.go:62: Failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:monitoring:kube-prometheus-exporter-kube-state" cannot list configmaps at the cluster scope

**Comment by metalmatze** [comment]
The Helm chart has moved to https://github.com/helm/charts/tree/master/stable/prometheus-operator please open a PR there.

**Pull request closed without merging**